.github/workflows/test.yml

```diff
@@ -0,0 +1,54 @@
+name: Test template creation
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+# Least privilege permissions
+permissions: read-all
+
+jobs:
+  test-copier:
+    runs-on: ubuntu-latest
+    env:
+      TEMPLATE_NAME: template-data-package
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+        with:
+          # Clone template to a subdirectory
+          path: ${{ env.TEMPLATE_NAME }}
+
+      - name: Set up uv
+        uses: astral-sh/setup-uv@bd01e18f51369d5a26f1651c3cb451d3417e3bba
+        with:
+          enable-cache: true
+          cache-dependency-glob: "**/uv.lock"
+
+      - name: Generate a data package
+        # TODO: Add other pre/post setup steps here.
+        run: |
+          # List options with `--data`
+          uvx copier copy --vcs-ref=HEAD ${{ env.TEMPLATE_NAME }} test-data-package \
+            --defaults \
+            --data project_name="My Project" \
+            --data project_description="." \
+            --data project_url="https://github.com/user/repo" \
+            --data author_name="John Smith" \
+            --data author_email="john@example.com"
```

Comment:
This won't work yet. Plus, it will need to be updated after we've converted things and added the variables in `copier.yaml`.